### ISSUE TITLE
Various wording questions/suggestions

### ISSUE THREAD

**Issue by jni** [emmanuelle/skimage-tomo-paper#11]
- [x] "synchrotron imaging campaign" — the word "campaign" here sounds weird to me. Is this standard terminology in this context? If not, I suggest "run" instead. Emma: yes, run sounds better. I'll make the change.
- [x] "raw data formats can be opened" — I actually wrote such a tool recently (https://github.com/jni/python-redshirt), and it was amazingly easy! I'm not sure how to write about/reference it, though, but would you like me to try to mention that? 
- [x] Move NumPy section before Jupyter/IDEs, since they are a more significant unifying thread. **Emma**: yes, will do.
- [x] Add Jupyter screenshot. **Emma**: can you do it? **Juan**: see #13 
- [x] Add sample code and figure for mayavi. **Emma**: will do.
- [x] Label Fig3 vertical panels with A, B, C, D, and use those for reference from within the text. Emma: OK.
- [ ] .(What are the per-population statistics on user visits? It's uninteresting to know that most people come from America and Asia. I'm curious to know whether scikit-image usage is high or low in Australia given the low population.) =)
- [x] Might be worth having a quick look through Google Scholar citations of our PeerJ paper to see whether there are specific tomography papers that we could cite as having used skimage.

**Comment by emmanuelle** [comment]
Thanks for the suggestions!

About the raw data reader, would you like to cite it as a reader for a specific raw data format? The problem with raw data is that every kind will require a specific treatment (which dimensions, header or no header) and that is it very difficult to guess anything about these parameters.

**Comment by jni** [comment]
Yeah, I know that's the problem, but the point that I'm trying to make, since you raised it in the text, is that if you do have the data format spec, it's super-easy to get the data into a numpy array using memmap. My reader is no masterpiece, but it does illustrate how one can make one quickly from the spec (which is linked from the readme in the repo). What do you think?

**Comment by jni** [comment]
let me know, e.g. by editing the first comment, whether you want to ignore some, handle some, or assign me some.

**Comment by jni** [comment]
So we don't both do the same thing. =)

**Comment by emmanuelle** [comment]
Sure, can you take care of the Jupyter screenshot? I'm just using it for teaching, so if you have a good real-life example, it will be better than a tutorial.

**Comment by emmanuelle** [comment]
What do you mean by per-population statistics? Number of visits for each country, divided by the total population? I'm afraid piwik does not provide such statistics. Of course it's possible to do it manually. If I retrieve a CSV file with visits per country, would you like to do the per-population computation?

**Comment by jni** [comment]
Yep! =)

**Comment by jni** [comment]
(Probably not for the paper. But I would consider removing the continental split from the paper.)

**Comment by emmanuelle** [comment]
Maybe these continental statistics are not necessary. I think I added them to make the part on user statistics longer, but it's not a good argument :-).

**Comment by emmanuelle** [comment]
I'll tick the different points as I push commits.

**Comment by emmanuelle** [comment]
For the continental statistics, what I found most interesting was that an important fraction of users come from developing countries (in particular India and China).

**Comment by emmanuelle** [comment]
I'm going to the swimming pool, should be back in 2 hours!

**Comment by emmanuelle** [comment]
Before it's too late in Melbourne (it's already late!!), a colleague who proofread the paper suggested to add a "current limitations and challenges" section. What would you point out as limitations and challenges   for processing 3-D images / large datasets?

**Comment by jni** [comment]
Oh that reminds me, I did think that mentioning xarray and Spark/Thunder
would be useful in this context! Both aim to solve the problem of
larger-than-RAM volumes (common with 3D data) and, because of skimage's
functional API, it integrates quite easily with both!

I think we still lack a solution that makes all that stuff perfectly
transparent, but it's coming. And that's very nice. =)

Of course you should mention the issue you raised in the skimage repo: that
there are still too make 2D-only functions that don't need to be.

**Comment by jni** [comment]
(Sorry for brevity, on phone still on way home.)

**Comment by emmanuelle** [comment]
thanks!

**Comment by jni** [comment]
btw this is what I meant by thunder if you don't know it:

http://docs.thunder-project.org/

**Comment by emmanuelle** [comment]
I'll have to submit the paper before you get up tomorrow morning, but thanks a lot for all your contributions!
